web/templates/settings/edit_invite_user_modal.hbs

```diff
@@ -0,0 +1,50 @@
+<form id="invite-user-form">{{ csrf_input }}
+    {{#if development_environment}}
+        <div class="alert" id="dev_env_msg"></div>
```

Comment:
Do we need this for multiuse-invites as well? I think it is only used to show that users can check the emails in development environment at `/emails`.

Reply:
Ah yes! This is not needed for multiuse-invites.